app/src/shared/modules/user/actions.js

```diff
@@ -34,14 +33,16 @@ import {
     DELETE_USER_ACCOUNT_SUCCESS,
     DELETE_USER_ACCOUNT_FAILURE,
 } from './constants'
-import routes from '$routes'
+import { clearStorage } from '$shared/utils/storage'
 
 // Logout
 export const resetUserData: ReduxActionCreator = createAction(RESET_USER_DATA)
 
 export const logout = () => (dispatch: Function) => {
+    clearStorage()
     dispatch(resetUserData())
-    dispatch(replace(routes.root()))
+
+    window.location.href = process.env.PLATFORM_ORIGIN_URL
```

Comment:
An attempt to solve [`PLARFORM-601`](https://streamr.atlassian.net/browse/PLATFORM-601).

Reply:
It should work unless React somehow steals the show and puts it through the Router. 

This will need a sanity check once it lands in Production as that's the only place to test it out.

Reply:
Agreed.